doc/content/compiletime.md

```diff
@@ -26,3 +26,19 @@ We're not exactly sure what the nature of the error is, but have
 [an issue](https://github.com/neams-th-coe/cardinal/issues/556)
 to track it. If this is affecting your workflow, please contact us so that
 we can reprioritize it.
+
+
+## Linking Errors After Updating Source
+
+Sometimes, despite following all instructions, `make -j8 MAKEFLAGS=-j8` will not work. For example, if it has been a while since updating Cardinal source, you have cleaned out `build/` and `install/`, and building causes a linking error
+
+```
+Linking libpng: -lpng16 -lz
+Linking Library cardinal/contrib/moose/framework/libmoose-opt.la...
+/usr/bin/ld: cannot find -lptscotchparmetis
+collect2: error: ld returned 1 exit status
+```
+
+Try running `git clean -xfd` from `cardinal/contrib/moose`.
+
+If all else fails, re-cloning is the last step to take. Make sure that all of your important branches exist elsewhere (perhaps on your remote fork or `neams-th-coe/cardinal`) and are up to date. Then you can delete `/path/to/cardinal`, clone again, and try a fresh build from the very first step; this should resolve any issues that are not environment issues. Remember to add your remote repositories again after re-cloning.
```

Comment:
I would just simplify this to say "If all else fails, clone cardinal in a new location and start over." We don't necessarily need people to delete what they have, they can test it out safely in a new location without wiping their current work. I think this will be a tiny bit safer :)